src/sensors/odometer/interrupt/DirectionalOdometer.cpp

```diff
@@ -77,7 +78,7 @@ long DirectionalOdometer::getDistance()
 
 float DirectionalOdometer::getSpeed()
 {
-    return DirectionlessOdometer::getSpeed() * static_cast<float>(getDirection());
+    return DirectionlessOdometer::getSpeed() * getDirection();
```

Comment:
:warning: **cppcoreguidelines-narrowing-conversions** :warning:
narrowing conversion from 'int' to 'float'